### ISSUE TITLE
How to change row header / label?

### ISSUE THREAD

**Issue by tomeroto** [jspreadsheet/ce#882]
I know that column can change the header text. But, 
How to change row header / label text?

**Comment by pphod** [comment]
Sorry, I am not sure if I understand your question. Can you rephrase, please?

**Comment by tomeroto** [comment]
To set column header we use "columns" properties.  How to set row header? Row header it has the order number of row default like 1, 2, 3... etc, I want to customize it, how?

**Comment by pphod** [comment]
If you talking about nestedHeaders, you must change those on initialization. There is no native method to change after the table is loaded.
